backend/alembic/versions/248a867312b4_add_branding_quality_guideline.py

```diff
@@ -0,0 +1,54 @@
+"""Add branding quality guideline
+
+Revision ID: 248a867312b4
+Revises: e87774655843
+Create Date: 2024-02-19 00:31:43.064367
+
+"""
+
+import datetime
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+from app.models import Guideline
+
+# revision identifiers, used by Alembic.
+revision = "248a867312b4"
+down_revision = "e87774655843"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.bulk_insert(
+        sa.table(
+            "guideline",
+            sa.Column("id", sa.String(), nullable=False),
+            sa.Column("url", sa.String(), nullable=False),
+            sa.Column("needed_to_pass_since", sa.Date(), nullable=False),
+            sa.Column("read_only", sa.Boolean(), nullable=False),
+            sa.Column("guideline_category_id", sa.String(), nullable=False),
+            sa.Column("show_on_fullscreen_app", sa.Boolean(), nullable=False),
+            sa.Column("order", sa.Integer(), nullable=False),
+        ),
+        [
+            {
+                "id": "general-has-primary-branding",
+                "url": "https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#brand-color",
```

Comment:
We likely want to redo the docs here and maybe change the headline and thus this link

Reply:
"Brand color" sounds fine to me :+1:

Reply:
I think it should be more self explanatory? It feels a bit displaced next to the other ones

Reply:
More like `has primary branding colors` (similar to `At least one screenshots`, that's the vibe I'm after)

Reply:
I think the appstream docs touch on that briefly, but we could too